packages/shared/src/dto/workflows/workflow-commons-fields.ts

```diff
@@ -64,6 +64,10 @@ export class WorkflowCommonsFields {
   @IsDefined()
   name: string;
 
+  @IsString()
+  @IsDefined()
+  triggerIdentifier: string;
```

Comment:
IMO we can refactor the legacy term and make it simplifier with `identifier` what do you think?